packages/@interaxyz/mobile/src/public/types.tsx

```diff
@@ -1,3 +1,5 @@
+import { ImageSourcePropType } from 'react-native'
```

Comment:
I see this is the only import into the types file for the framework so far. Could this result in circular imports? Maybe the below import would work better, but I am not sure. Might want to check with @Jean or @kathaypacific.
```suggestion
import type { ImageSourcePropType } from 'react-native'
```

Reply:
added the type

Reply:
Both are ok because it's an external dependency.

Reply:
importing from our deps is safe :) the cycles are only a problem if we are importing from other files from our own project (e.g. statsig/index imports from src/web3/selectors which then imports from statsig/index) - there won't be a cycle here even if this file is then imported into other files, because we only import from react-native, react-native does not import from us

Reply:
Thank you, @kathaypacific @jeanregisser for the explanations!